src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -114,6 +126,43 @@ public static void removeBiome(BiomeType type, BiomeEntry entry)
         }
     }
 
+    /**
+     * Adds an OceanBiomeEntry to a type of ocean.
+     * This method should be called during mod loading to add a new OceanBiomeEntry to a type of ocean (e.g. a new Frozen Ocean biome).
+     * This method is threadsafe.
+     *
+     * @param type The type of ocean for this OceanBiomeEntry.
+     * @param entry The OceanBiomeEntry to add.
+     */
+    public static void addOceanBiome(OceanType type, OceanBiomeEntry entry)
+    {
+        synchronized (BiomeManager.class)
+        {
+            forgeOceanBiomes[type.ordinal()].add(entry);
+        }
+    }
+
+    /**
+     * Removes an OceanBiomeEntry for a type of ocean.
+     * This method should be called to remove a OceanBiomeEntry from a type of ocean.
+     * This method is threadsafe.
+     *
+     * @param type The type of ocean for this OceanBiomeEntry.
+     * @param entry The OceanBiomeEntry to remove.
+     */
+    public static void removeOceanBiome(OceanType type, OceanBiomeEntry entry)
+    {
+        int idx = type.ordinal();
+        synchronized (BiomeManager.class)
+        {
+            if(type == OceanType.NORMAL && forgeOceanBiomes[idx].size() <= 1)
+            {
+                throw new IllegalStateException("At least one NORMAL type ocean biome entry must be present");
+            }
+            forgeOceanBiomes[idx].remove(entry);
```

Comment:
This requires you to implement `equals` and `hashCode` in `OceanBiomeEntry`, otherwise this method is hard to use (you have to call `getOceanBiomes` first and find the entry you want to remove there...).
Your size check also does not take into account the fact that one might pass in an entry that is not actually contained in the list.

Reply:
Hm yeah, those are true. 
Although, I'm thinking of removing this method so that there'll be fewer conflict issues in mods, thoughts?